### DISCUSSION TITLE
Mapserver

### DISCUSSION THREAD

**Discussion by amigators** [Homebrew/discussions#2359] [Getting started] [answered]
Is there anything I can read or review about the install process/procedure for getting Mapserver up and running.

**Comment by SMillerDev** [comment]
https://mapserver.org/

**Comment by amigators** [comment, marked as answer]
Thank you. Have made some progress. Apache is running (using Safari) and mapserver is running based on terminal access.

I started apache since it is installed already on my Mac.
I used Homebrew to install the Mapserver.

I seem to be getting a permissions error when I try to access mapserv

I have accessed Perl without any permissions issues (the  little code that prints Hello World). 
I believe I have tried both access alias mapserv and the actual mapserv both with permissions denied.  I have tried setting permissions from the terminal app and I believe it indicates that the permissions are consistent with the documentation.


Any suggestions?
